landlab/components/soil_grading/soil_grading.py

```diff
@@ -209,14 +208,15 @@ def __init__(
         self._initial_median_size = initial_median_size
         self._CV = CV
         self._initial_total_soil_weight = initial_total_soil_weight
-        if std == None:
+        if std is None:
             std = self._initial_median_size * self._CV
         self._std = std
         self._is_bedrock_distribution_flag = is_bedrock_distribution_flag
         self._precent_of_volume_in_spread = precent_of_volume_in_spread
 
         # Create out fields.
-        # Note: Landlabs' init_out_field procedure will not work for the 'grains__weight' and 'grains_fractions__size' fields
+        # Note: Landlabs' init_out_field procedure will not work
+        # for the 'grains__weight' and 'grains_fractions__size' fields
```

Comment:
Why are the fields configured with 2-3 dimensions instead of 1-2? That is, why isn't median_size__weight just number_of_nodes, and grains_fractions__size (for example) just number_of_nodes x number_of_size_classes?

Reply:
Oops, no reason. Thanks. I will change it accordingly.